### DISCUSSION TITLE
Challenges with Publishing Website on GitHub Pages and DNS Check Delay

### DISCUSSION THREAD

**Discussion by avimanyurimal** [community/community#56608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I encountered a frustrating issue while trying to publish my website on GitHub Pages. After configuring the domain through Cloudflare and following all the necessary steps, my website remained inaccessible for over 40 hours due to an ongoing DNS check in progress. Despite Cloudflare confirming the domain's activation, GitHub Pages failed to complete the DNS check. I needed guidance to resolve the DNS issue and ensure that my website could finally go live. The delay and lack of responsiveness from GitHub caused significant inconvenience and hindered my website deployment process.
![Screenshot 2023-05-28 162012](https://github.com/community/community/assets/127291200/778ae77a-6a8b-4249-80d4-373b82927d01)

**Comment by lemerien** [comment]
Same issue here ,what's going on with the activation !
